farmbot_testing.py

```diff
@@ -0,0 +1,161 @@
+import unittest
+import json
+from funct_CLASSES import Farmbot
```

Comment:
I'm getting an error: `ModuleNotFoundError: No module named 'funct_CLASSES'`. Should this be `from farmbot_utilities import Farmbot`?